.github/workflows/publish-to-test-pypi.yml

```diff
@@ -0,0 +1,106 @@
+name: Publish Python 🐍 distribution 📦 to TestPyPI
+
+on:
+  workflow_dispatch:  # Keep manual trigger
+    inputs:
+      rc_version:
+        description: 'RC version number (e.g., 1, 2, 3)'
+        required: true
+        type: string
+  schedule:
+    - cron: "0 0 * * *"  # Run every day at midnight
+
+jobs:
+  trigger-client-and-models-build:
+    name: Trigger llama-stack-client and llama-models build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get date
+      id: date
+      run: echo "date=$(date +'%Y%m%d')" >> $GITHUB_OUTPUT
+    - name: Compute version based on dispatch event
+      id: version
+      run: |
+        # Read base version from pyproject.toml
+        if [ "${{ github.event_name }}" = "schedule" ]; then
+          echo "version=.dev${{ steps.date.outputs.date }}" >> $GITHUB_OUTPUT
+        elif [ "${{ github.event_name }}" = "workflow_dispatch" ]; then
+          echo "version=rc${{ inputs.rc_version }}" >> $GITHUB_OUTPUT
+        else
+          echo "version=.dev$(shuf -i 10000000-99999999 -n 1)" >> $GITHUB_OUTPUT
+        fi
+    - name: Trigger llama-stack-client workflow
+      uses: peter-evans/repository-dispatch@v2
+      with:
+        token: ${{ secrets.PAT_TOKEN }}
+        repository: meta-llama/llama-stack-client-python
+        event-type: build-client-package
+        client-payload: '{"source": "llama-stack-nightly", "version": "${{ steps.version.outputs.version }}"}'
```

Comment:
I think this can be replaced by something like if you'd like to get rid of the dependency on `peter-evans/repository-dispatch`:
```
          curl -X POST https://api.github.com/repos/meta-llama/llama-stack-client-python/dispatches \
          -H 'Accept: application/vnd.github.everest-preview+json' \
          -H "authorization: Bearer ${{ secrets.PAT_TOKEN }}" \
          --data '{"event_type": "build-client-package", "client_payload": xxxxxx}"}}'
```

Reply:
Thanks for the suggestion! Updated to use curl